basis/io/directories/unix/unix.factor

```diff
@@ -67,7 +67,8 @@ M: unix copy-file ( from to -- )
 
 : >directory-entry ( dirent* -- directory-entry )
     [ d_name>> alien>native-string ]
-    [ d_type>> dirent-type>file-type ] bi
+    [ drop f ] ! TODO: set size slot
```

Comment:
If so, I could put the size slot in the windows-directory-entry. Would that be better?